src/api/frame/frame.ts

```diff
@@ -18,6 +19,12 @@ export interface FrameInfo {
  */
 // tslint:disable-next-line
 export default class _FrameModule extends Base {
+    public frameIdentity: any;
+    constructor(wire: Transport) {
+        super(wire);
+        this.frameIdentity = fin.__internal_.entityInfo;
```

Comment:
Can you move this part to environment as i mentioned above.  So it will not break when we run it in node environment.

Reply:
Fixed :+1: